src/components/skills-quiz/SearchJobCard.jsx

```diff
@@ -7,20 +7,22 @@ import { SkillsContext } from './SkillsContextProvider';
 import { SET_KEY_VALUE } from './data/constants';
 import JobCardComponent from './JobCardComponent';
 
-const SearchJobCard = ({ index }) => {
+function SearchJobCard({ index }) {
   const { refinements } = useContext(SearchContext);
   const { name: jobs } = refinements;
   const [isLoading, setIsLoading] = useState(true);
   const { dispatch, state } = useContext(SkillsContext);
   const { interestedJobs } = state;
-  const jobsToFetch = useMemo(() => {
-    const jobsArray = [];
-    if (jobs) {
-      jobs.forEach(job => jobsArray.push(`name:${job}`));
-    }
-    return jobsArray;
-  },
-  [jobs]);
+  const jobsToFetch = useMemo(
+    () => {
+      const jobsArray = [];
+      if (jobs) {
+        jobs.forEach(job => jobsArray.push(`name:${job}`));
+      }
+      return jobsArray;
+    },
+    [jobs],
```

Comment:
[curious] Is `jobsToFetch` re-computed on every render of `SearchJobCard `? I believe `jobs` in the dependency array will have a new variable reference on each re-render, and thus cause `jobsToFetch` to unnecessarily re-compute each re-render.

Reply:
I do believe that you are right here, with `jobs` in the dependency array, it seems that jobsToFetch will re-compute with each re-render.